docs/concepts.md

```diff
@@ -50,9 +53,325 @@ Dynamic provisioning is when an resource claim does not find a matching resource
 The newly provisioned resource is automatically bound to the resource claim.
 To enable dynamic provisioning the administrator needs to create one or more resource class objects.
 
-## Connection Secrets
-
-Workloads reference all the resources the consume in their `resources` section.
+### Connection Secrets
+Workloads reference all the resources they consume in their `resources` section.
 This helps Crossplane setup connectivity between the workload and resource, and create objects that hold connection information.
 For example, for a database provisioned and managed by Crossplane, a secret will be created that contains a connection string, user and password.
 This secret will be propagated to the target cluster so that it can be used by the workload.
+
+### Secure Connectivity
+networks, subnets, peering, NACLs,etc.
+
+# Glossary
+
+## Kubernetes
+Crossplane is built on the Kubernetes API machinery as a platform for declarative management,
+and common terminology used from the [Kubernetes Glossary][kubernetes-glossary] where possible,
+and we don't seek to reproduce that glossary here.
+
+[kubernetes-glossary]: https://kubernetes.io/docs/reference/glossary/?all=true
+However we'll summarize some commonly used concepts for convenience.
+
+### CRD
```

Comment:
Should we use CRD or Custom Resource Definition as the heading? The former is probably more recognisable while the latter is probably more correct.

Reply:
I just want to #crd

Reply:
but happy to swap them around if y'all want to #custom-resource-definition

Reply:
I don't feel strongly.